cpp/include/cudf/aggregation.hpp

```diff
@@ -147,7 +185,8 @@ class rolling_aggregation : public virtual aggregation {
 
  protected:
   rolling_aggregation() {}
-  rolling_aggregation(aggregation::Kind a) : aggregation{a} {}
+  /// constructor inherited from cudf::aggregation
+  using aggregation::aggregation;
```

Comment:
`using declaration` style here instead of old code. 
This is the only code change in this PR.